.packit.yaml

```diff
@@ -37,15 +37,9 @@ jobs:
   - <<: *copr
     project: qm
     targets:
-      fedora-40-aarch64: {}
-      fedora-40-ppc64le: {}
-      fedora-40-x86_64: {}
-      fedora-39-aarch64: {}
-      fedora-39-ppc64le: {}
-      fedora-39-x86_64: {}
-      fedora-38-aarch64: {}
-      fedora-38-ppc64le: {}
-      fedora-38-x86_64: {}
+      fedora-stable-aarch64: {}
+      fedora-stable-ppc64le: {}
+      fedora-stable-x86_64: {}
```

Comment:
I assume the architectures are listed here to avoid building on `s390x`, right?

Reply:
yes